config/discourse_defaults.conf

```diff
@@ -390,8 +390,14 @@ log_line_max_chars = 160000
 # this value is included when generating static asset URLs.
 # Updating the value will allow site operators to invalidate all asset urls
 # to recover from configuration issues which may have been cached by CDNs/browsers.
-asset_url_salt = 
+asset_url_salt =
 
 # disable service-worker caching. An 'offline' page will still be used as an offline fallback.
 # This flag is for a temporary experiment, and may be removed at any time
 disable_service_worker_cache = true
+
+# Sets cache sizes. This many sites can have the maximum number of keys in each LiveCache
+live_cache_multiplier = 200
+
+# false means DistributedCache, true means LiveCaches are used for in-process caches
+use_live_caches = false
```

Comment:
I don't think there is need for this to be a global setting because once we add something in here we want to assume it becomes public API. We can achieve the same outcome with a hidden site setting.

Reply:
ok